OWML.Launcher/App.cs

```diff
@@ -99,6 +93,7 @@ private void ShowModList(IList<IModData> mods)
             foreach (var modData in mods)
             {
                 var stateText = modData.Config.Enabled ? "" : "(disabled)";
+
```

Comment:
huh?

Reply:
left over from when I was changing all this around - fixed